networks/README.md

```diff
@@ -0,0 +1,46 @@
+# Networks
+
+## Overview
+
+With this example we demonstrate how to create a single network as well as two networks with one stack.
```

Comment:
@dschwabeS11 what is the use case? which problem does this solve? Are there downsides? I think that should be part of the description.

Reply:
The difference is that this one does only spawn network components without hosts and that is contains an example for a single and multiple networks.
The other example might have to be renamed. We just started to clean up the repo but don't want to remove old content before there is new content that can replace it.

---

Will extend the overview.

Reply:
@stefreak Extended overview text. Please review again.

Reply:
👍